api/service/networkinfo/service.go

```diff
@@ -132,6 +132,7 @@ func (s *Service) Init() error {
 	utils.Logger().Info().Str("Rendezvous", string(s.Rendezvous)).Msg("Announcing ourselves...")
 	s.discovery = libp2pdis.NewRoutingDiscovery(s.dht)
 	libp2pdis.Advertise(ctx, s.discovery, string(s.Rendezvous))
+	libp2pdis.Advertise(ctx, s.discovery, string(p2p.GroupIDBeaconClient))
```

Comment:
@harmony-ek Why do we advertise on BeaconClient group here?

Reply:
@rlan35, that is because beaconchain now sends new blocks to the network using `node.BroadcastNewBlock()`, which sends the block to the client group of the shard.  See #1391 for details.